client/src/components/Gallery.jsx

```diff
@@ -27,6 +27,27 @@ class Gallery extends React.Component {
     this.handleSliderClick = this.handleSliderClick.bind(this);
   }
 
+  componentDidMount() {
+    axios
+    .get('/restaurants/100/photos')
+    .then((response) => {
```

Comment:
Don't hard code the restaurant id, randomly generate an id from 1 to 100.

Reply:
ok I can change it